### ISSUE TITLE
Fails to compile with up-to-date-dependencies

### ISSUE THREAD

**Issue by alaendle** [hyraxbio/hyraxAbif#8]
Fails in conjunction with current stackage nightly package set (using ghc-9.6):

```
/home/curators/work/unpack-dir/unpacked/hyraxAbif-0.2.4.4-54aed6cecc4c5153137af703c13e70fb097b55be6b70810501522bef5e37e48c/test/Generators.hs:17:35: error: [GHC-39999]
           • Ambiguous type variable ‘f0’ arising from a use of ‘Gen.element’                                                        prevents the constraint ‘(Foldable f0)’ from being solved.                                                              Probable fix: use a type annotation to specify what ‘f0’ should be.                                                     Potentially matching instances:
               instance Foldable (Either a) -- Defined in ‘Data.Foldable’                                                              instance Foldable Identity -- Defined in ‘Data.Functor.Identity’                                                        ...plus 9 others                                                                                                        ...plus 34 instances involving out-of-scope types                                                                       (use -fprint-potential-instances to see them all)                                                                   • In the second argument of ‘Gen.text’, namely                                                                              ‘(Gen.element "ACGTMRWSYKVHDBNX")’                                                                                    In the expression:
               Gen.text (Range.linear 1 1000) (Gen.element "ACGTMRWSYKVHDBNX")                                                       In an equation for ‘nucsGen’:
                 nucsGen
                   = Gen.text (Range.linear 1 1000) (Gen.element "ACGTMRWSYKVHDBNX")                                              |                                                                                                                    17 |   Gen.text (Range.linear 1 1000) (Gen.element "ACGTMRWSYKVHDBNX")                                                     |                                   ^^^^^^^^^^^
```

@andrevdm

**Comment by andrevdm** [comment]
Fixed in hyraxAbif-0.2.4.5

(tests were breaking)

**Comment by alaendle** [comment]
Thanks; will re-enable the package soon.
